src/config/dynamic-theme-fixes.config

```diff
@@ -19486,6 +19486,15 @@ CSS
 
 ================================
 
+newreleases.io
+
+CSS
+.bg-white > .bg-cover {
+    background-image: none !important;
```

Comment:
maybe instead of removing whole background image we will mix it with background that will be dark

Check how it can look with this part of code:
```
    background-blend-mode: multiply !important;
    background-color: ${rgba(255,255,255,0.95)} !important;
```